<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Free Form Drawing (By Sokhavirith Sorn)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tools</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_pencil">
           <property name="styleSheet">
            <string notr="true">background-color : red</string>
           </property>
           <property name="text">
            <string>Pencil</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/Icons/Icons/paint.png</normaloff>:/Icons/Icons/paint.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_line">
           <property name="styleSheet">
            <string notr="true">background-color : #bfbfbf</string>
           </property>
           <property name="text">
            <string>Line</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/Icons/Icons/Line.png</normaloff>:/Icons/Icons/Line.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_circle">
           <property name="styleSheet">
            <string notr="true">background-color : #bfbfbf</string>
           </property>
           <property name="text">
            <string>Circle</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/Icons/Icons/circle.png</normaloff>:/Icons/Icons/circle.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_rect">
           <property name="styleSheet">
            <string notr="true">background-color : #bfbfbf;

</string>
           </property>
           <property name="text">
            <string>Rect</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/Icons/Icons/rec.png</normaloff>:/Icons/Icons/rec.png</iconset>
           </property>
           <property name="autoRepeatInterval">
            <number>98</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_triangle">
           <property name="styleSheet">
            <string notr="true">background-color : #bfbfbf</string>
           </property>
           <property name="text">
            <string>Triangle</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/Icons/Icons/triangle.png</normaloff>:/Icons/Icons/triangle.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sb_size">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clear">
           <property name="styleSheet">
            <string notr="true">background-color : #bfbfbf</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/Icons/Icons/clear.png</normaloff>:/Icons/Icons/clear.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Exit</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../exit.png</normaloff>../exit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GLWidget" name="GLwidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClear_Screen">
   <property name="text">
    <string>Clear Screen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
